reflex-common/reflex-common/src/main/python/mcenter_cli/parallelm/mcenter_cli/list_actions.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+
+def list_actions(mclient, args):
+    success = None
+    if args.success:
+        success = True
+    if args.failed:
+        success = False
+    print("Getting action entries for: ")
+    print("    Object: " + str(args.object))
```

Comment:
you can write as `print("    Object: {}".format(args.object))`

Reply:
Fixed